assets/src/edit-story/components/colorPicker/colorPicker.js

```diff
@@ -167,6 +175,7 @@ ColorPicker.propTypes = {
   hasGradient: PropTypes.bool,
   hasOpacity: PropTypes.bool,
   color: PatternPropType,
+  addActions: PropTypes.oneOfType([PropTypes.bool, PropTypes.func]),
```

Comment:
What will it do as a bool, when it's always [used as a function in line 165](https://github.com/google/web-stories-wp/pull/753/files#diff-9eae147e026beb102527eb1a924fc5c7R165)?

Reply:
If it's `false` then actions aren't displayed, that was the idea for the bool. If it should be set to `false` for some reason somewhere. Might be redundant 😝

Reply:
But if it's `true`, it'll throw. So just allowing `func` and default'ing to `null` is probably safest.